sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -659,8 +659,12 @@ CREATE UNIQUE INDEX `token` ON `api_token` (`token`);
 CREATE UNIQUE INDEX `pid` ON `patient_access_onsite` (`pid`);
 #EndIf
 
+#IfNotColumnTypeDefault form_encounter uuid binary(16) NULL
+ALTER TABLE `form_encounter` DROP `uuid`;
+#EndIf
```

Comment:
Above block not needed. I had to add this for the patient_data column for those that are using the master branch in production (since i previously added a column that didn't default to null, I had to add the block to fix that and convert it to NULL). Since the form_encounter uuid has not yet been added to the codebase, you don't need to do this :)

Reply:
Yea Agree, It was needed only once for my local but not needed here :+1: